sdk/servicebus/service-bus/src/receivers/receiverCommon.ts

```diff
@@ -293,6 +318,22 @@ export async function retryForever<T>(
   retryFn: typeof retry = retry
 ): Promise<T> {
   let numRetryCycles = 0;
+  const config = args.retryConfig;
```

Comment:
I duplicated the code from core-amqp's retry as they are used internally and not really worth to expose to reuse.